docs/phdata.rst

```diff
@@ -232,16 +261,33 @@ beam-splitter the fields:
 
 - **split_ch1**
 - **split_ch2**
+- etc..
 
 specify which detector pixel is used in each of the "beam-split" channels.
 
-All previous fields are arrays containing one or more :ref:`detector_ids`.
-For example, a 2-color smFRET measurement will have only one value in
-``spectral_ch1`` (donor) and one value in ``spectral_ch2``
-(acceptor). A 2-color smFRET measurement with polarization
+If :ref:`marker_ids` are also recorded, then the fields (*new in v0.5*):
+
+- **markers1**
+- **markers2**
+- etc...
+
+are used to categorize each marker id. Currently there are no specifications
+for the type of markers. These should be explained in
+the :ref:`/user/experimental_settings<exp_settings>` group for human readable
+understanding enabling custom code to be written to process these markers.
+All official software for v0.5 will ignore data points specified as markers.
+Future releases may specify sorts of measurements that will have defined uses
+for particular markers.
+
+All previous fields are arrays containing one or more :ref:`record IDs<record_ids>`
+(Detector IDs for all ``spectral_chX``, ``polarization_ch1`` and ``split_chX``,
+and :ref:`marker IDs <marker_ids>` for ``markersX``).
+For example, a 2-color smFRET measurement without polarization or split channels
+(2 detectors) will have only one value in ``spectral_ch1`` (donor) and one value in
+``spectral_ch2`` (acceptor). A 2-color smFRET measurement with polarization
```

Comment:
I don't think this part applies to the markers (non-photons), so perhaps this paragraph should come before the markers (and not mention markers).

(And perhaps a single field, `marker_ids` (or `non_photon_ids`) is sufficient, because we do not anticipate these forming an axis in an N-dimensional space in the way that the spectral/pol/split ones do.)